examples/createLayer_custom_UI_functions.html

```diff
@@ -0,0 +1,293 @@
+          <a href="#close" class="cartodb-popup-close-button close">x</a>
+           <div class="cartodb-popup-content">
+            <div class="content" style="padding:20px">
+              <h3>Country:</h3> 
+              <p>{{content.data.adm0name}}</p>
+              <h3>City:</h3>
+              <p>{{content.data.name}}</p>
+              <h3>Max population:</h3>
+              <p>{{content.data.pop_max}}</p>
+              <div>
+          <div>
+          <div class="cartodb-popup-tip-container"></div>
+        </div>
+    </script>
+
+<script>
+  function main() {
+    
+    var lat;
+    var lon;
+
+    // define map object
+    var map = new L.Map('map', {
+      zoomControl: false,
+      center: [43, 0],
+      zoom: 3
+
+    });
+  
+    /* Define CartoDB layer with createLayer(). More information here: 
+
+    http://docs.cartodb.com/cartodb-platform/cartodb-js/api-methods/#cartodbcreatelayermap-layersource--options--callback */
+    cartodb.createLayer(map, {
+      user_name: 'oboix',
+      type: 'cartodb',
+      sublayers: [
+        {
+          type: "http",
+          urlTemplate: "http://{s}.basemaps.cartocdn.com/dark_nolabels/{z}/{x}/{y}.png",
+          subdomains: [ "a", "b", "c" ]
+        },
+        {
+           type: "mapnik",
+           sql: 'select * from populated_places_spf',
+           cartocss: '#populated_places_spf[adm0name = "Spain"]{ marker-fill: #fbb4ae; marker-allow-overlap: true;}#populated_places_spf[adm0name = "Portugal"]{ marker-fill: #ccebc5; marker-allow-overlap: true;}#populated_places_spf[adm0name = "France"]{ marker-fill: #b3cde3; marker-allow-overlap: true;}',
+           interactivity: ['cartodb_id','adm0name','name','pop_max']
+         
+        },
+        {
+          type: "http",
+          urlTemplate: "http://{s}.basemaps.cartocdn.com/dark_only_labels/{z}/{x}/{y}.png",
+          subdomains: [ "a", "b", "c" ]
+        }
+      ]
+    })
+  .addTo(map) // add cartodb layer and basemap to map object
+  .done(function(layer) {
+
+    /*Once that the basemap and cartodb layer are added....*/
+      
+        /* Enable mouse events with CartoDB layers */
+
+        // 1- Set CartoDB layer interaction
+        layer.setInteraction(true);
+
+        // 2- Custom mouseover event on CartoDB layers
+        layer.on('featureOver',function(e,latlng,pos,data){
+          lat = (latlng[0]).toFixed(2) // show latitude of clicked point in the console
+          lon = (latlng[1]).toFixed(2) // show longitude of clicked point in the console
+          
+        // 3- Add coordinates of the selected geometry in the div element 
+        //  with id = "box" 
+           document.getElementById("lat").innerHTML = lat;
+           document.getElementById("lon").innerHTML = lon;
+        });
+
+      /* Set custom infowindow (= click infowindow) More information here:
+      http://docs.cartodb.com/cartodb-platform/cartodb-js/ui-functions/#cartodbvisvisaddinfowindowmap-layer-fields--options
+
```

Comment:
I'd try to format/indent these comments a bit better. For example: remove this☝️ blank line.